.github/workflows/run-integ-tests.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
           set -x
           cd _test_resources
           npm i
-          pulumi -s integ-test-shared stack select --create
+          pulumi -s integ-test-shared-${{ env.AWS_REGION }} stack select --create
```

Comment:
IMO we shouldn't use region as proxy for separating different actions and instead use some input that can be kept constant but separate between the two repos.

Reply:
i could use the sample apps repo name instead if you would prefer that. 

The only benefit i see with region is pro and oss have the same exact resources in the shared stack, so if we did deploy them in the same region is less running services = lower cost. That may not be the case forever though if those tests change, but my hunch is they will always support the same shared resources which is why i opted for region

Reply:
Ah, if we're intending pro and oss to share then that makes sense. 👍